include/pokemon.h

```diff
@@ -169,13 +169,13 @@ BOOL Mon_CanUseGracidea(Pokemon *mon);
 void Party_ResetAllShayminToLandForm(Party *party);
 BOOL Party_TryResetShaymin(Party *party, int min_max, const RTCTime *time);
 BOOL Mon_UpdateRotomForm(Pokemon *mon, int form, int defaultSlot);
-void sub_02071FDC(SOUND_CHATOT *r6, u32 r5, u16 r4, s32 unused, s32 sp18, u32 sp1C, u32 sp20, u32 sp24);
-void sub_0207204C(SOUND_CHATOT *r7, u32 r6, u16 r5, s32 r4, s32 sp20, u32 sp24, u32 sp28, u32 sp2C, u8 sp30);
-void sub_020720D4(Pokemon *mon);
-void sub_020720FC(Pokemon *mon, PlayerProfile *a1, u32 pokeball, u32 a3, u32 encounterType, enum HeapID heapID);
-void sub_0207213C(BoxPokemon *boxMon, PlayerProfile *playerProfile, u32 pokeball, u32 a3, u32 encounterType, enum HeapID heapID);
-void sub_0207217C(Pokemon *mon, PlayerProfile *a1, u32 pokeball, u32 a3, u32 encounterType, enum HeapID heapID);
-void sub_02072190(BoxPokemon *boxMon, PlayerProfile *a1, u32 pokeball, u32 a3, u32 encounterType, enum HeapID heapID);
+void Mon_ChatterPlayCryEx(SOUND_CHATOT *soundChatot, u32 r5, u16 species, s32 unused, s32 sp18, u32 sp1C, u32 sp20, u32 sp24);
+void Mon_ChatterPlayCryVariant(SOUND_CHATOT *soundChatot, u32 r6, u16 species, s32 r4, s32 sp20, u32 sp24, u32 sp28, u32 sp2C, u8 sp30);
+void PlayMonCry(Pokemon *mon);
```

Comment:
Pokemon_PlayCry

Reply:
bit of inconsistiency between Mon_ and Pokemon_
the struct is called Pokemon so all the Mon_ should be Pokemon_